airflow/config_templates/config.yml

```diff
@@ -500,6 +500,14 @@ core:
       type: integer
       example: ~
       default: "4096"
+    enable_purging_stale_dags:
```

Comment:
```suggestion
    auto_purge_stale_dags:
```

Or maybe just `purge_stale_dags` would be enough.

Reply:
Ok, i prefer  purge_stale_dags